hack/ginkgo-e2e.sh

```diff
@@ -163,6 +163,18 @@ if [[ "${GINKGO_NO_COLOR}" == "y" ]]; then
   ginkgo_args+=("--no-color")
 fi
 
+if [[ "${E2E_REPORT_DIR:-}" ]]; then
```

Comment:
don't you use `[[ -n "${E2E_REPORT_DIR:-}" ]] ?

Reply:
Both are equivalent. Let me add  `-n` if that is more familiar => https://github.com/kubernetes/kubernetes/compare/b7e6ff2dacdb4c1bf49f82cc9d810af5250e4052..cbb6ea5ca462b9de664d0eba9a0495847537408c